packages/clarity-js/src/data/metadata.ts

```diff
@@ -310,7 +311,8 @@ function user(): User {
     if (parts.length > 2) { output.expiry = num(parts[2], 36); }
     // Check if we have explicit consent to track this user
     if (parts.length > 3 && num(parts[3]) === 1) { output.consent = BooleanFlag.True; }
-    if (parts.length > 4 && num(parts[1]) > 1) { output.dob = num(parts[4]); }
+    if (parts.length > 4 && num(parts[4]) === 1) { output.ad_consent = BooleanFlag.True; }
```

Comment:
we should not do this.  new parts should be at the end or there wont be any backward compatibility